### ISSUE TITLE
Cannot patch models that contain passwords without passing a password

### ISSUE THREAD

**Issue by jmeyers91** [scoutforpets/objection-password#8]
I'm trying to implement a way for users to update their profile in an app I'm working on, but if I try to call `.patch` without passing a password value, I get this error:

```
Unhandled Error: password must not be empty
        at Object.generateHash (/Users/james/projects/stem_familia/Server/node_modules/objection-password/index.js:74:27)
        at Promise.resolve.then (/Users/james/projects/stem_familia/Server/node_modules/objection-password/index.js:38:33)
        at <anonymous>
    From previous event:
        at runCallback (timers.js:800:20)
        at tryOnImmediate (timers.js:762:5)
        at processImmediate [as _immediateCallback] (timers.js:733:5)
    From previous event:
        at builder._operations.forEach.op (/Users/james/projects/stem_familia/Server/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1066:25)
        at Array.forEach (<anonymous>)
        at beforeExecute (/Users/james/projects/stem_familia/Server/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1064:23)
        at Promise.try (/Users/james/projects/stem_familia/Server/node_modules/objection/lib/queryBuilder/QueryBuilder.js:518:30)
    From previous event:
        at QueryBuilder.execute (/Users/james/projects/stem_familia/Server/node_modules/objection/lib/queryBuilder/QueryBuilder.js:518:23)
        at QueryBuilder.then (/Users/james/projects/stem_familia/Server/node_modules/objection/lib/queryBuilder/QueryBuilder.js:425:26)
```

How am I supposed to patch a model that contains a password without requiring a new password is passed?

It looks like [this PR](https://github.com/scoutforpets/objection-password/pull/7) fixes this issue.

**Comment by jamesdixon** [comment]
@jmeyers91 yes, looks like that PR resolves the issue. Would you mind giving that PR a try and if it works, I'll merge? Thanks!

**Comment by jmeyers91** [comment]
Sure I'll test it out.

**Comment by jamesdixon** [comment]
Appreciated! I'm no longer on the project I created this add-on for, so this helps a lot 👍

**Comment by jmeyers91** [comment]
It works fine in my project. I'm able to patch with or without passwords. When a password is passed, it appears to be hashing correctly.

**Comment by jamesdixon** [comment]
@jmeyers91 thanks. I've merged it to master. I'll publish to npm as soon as I have a few

**Comment by jamesdixon** [comment]
@jmeyers91 published as 1.0
